src/components/screen/Setting/ChangePwView.tsx

```diff
@@ -0,0 +1,96 @@
+import { Alert, KeyboardAvoidingView, SafeAreaView } from 'react-native';
+import {
+  DefaultTheme, withTheme,
+} from 'styled-components/native';
+import { InnerContainer, ModalCloseButton, ModalHeader, ModalTitle, StyledTextInput } from './styles';
+import React, { useState } from 'react';
+
+import Button from '../../shared/Button';
+import { Ionicons } from '@expo/vector-icons';
+import { getString } from '../../../../STRINGS';
+
+export interface Props {
+    theme: DefaultTheme;
+    close(): void;
+    checkCurrentPw(): Promise<boolean>;
+}
+
+function ChangePwView(props: Props): React.ReactElement {
+  const { theme, close, checkCurrentPw } = props;
+  const [currentPwVerified, setCurrentPwVerified] = useState(false);
+  const [currentPw, setCurrentPw] = useState('');
+  const [newPw, setNewPw] = useState('');
+  const [newPwCheck, setNewPwCheck] = useState('');
+  const checkCurrent = (): void => {
+    checkCurrentPw().then((verified) => {
```

Comment:
The naming `check` means A, B or more.
And this function has two roles `Updating state` and `Validating password`.
So, if the result value is `boolean`, the roles should be separated for readability and intuition.

For example,
```js
isValid~()  // Validate password
updated~Verified(isValid~()) // Update state
```

It makes code clear. (ref. [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single_responsibility_principle))